src/html/popup.html

```diff
@@ -45,6 +45,14 @@
 
         <main>
             <div class="container">
+                <div id = "rateSection">
+                    <div class="section">
```

Comment:
Once user clicks on "Rate it now" button, the ' div id = "rateSection" ' and ' div class="divider" ' should be hidden. rateSection div wraps the above two divs